app-web/gatsby-config.js

```diff
@@ -70,6 +71,7 @@ module.exports = {
       options: {
         tokens: {
           GITHUB_API_TOKEN: process.env.GITHUB_TOKEN,
+          SOURCE_REGISTRY_TYPE: process.env.REGISTRY_TYPE || 'SourceRegistryYaml',
```

Comment:
I think this should be moved out of tokens and be stand alone

Reply:
also could we should perhaps document this somewhere just so we know how the source registry is acutally sourced. Its not too important now, but if this is ever exported as an npm package we will need explain the dependancy on gatsby source filesystem and gatsby transformer yaml plugins